library/ix-dev/community/minecraft/questions.yaml

```diff
@@ -95,44 +95,46 @@ questions:
                 description: Spigot
               - value: BUKKIT
                 description: Bukkit
-              - value: FORGE
-                description: Forge
-              - value: CATSERVER
-                description: CatServer
-              - value: CRUCIBLE
-                description: Crucible
-              - value: AUTO_CURSEFORGE
-                description: Auto CurseForge
-              - value: CUSTOM
-                description: Custom
+              - value: PAPER
+                description: Paper
+              - value: FOLIA
+                description: Folia
               - value: FABRIC
                 description: Fabric
-              - value: FTBA
-                description: Feed The Beast
               - value: FORGE
                 description: Forge
-              - value: LOLISERVER
```

Comment:
Yes LOLISERVER was removed as far as I can tell. But it should indeed still work on older versions. So I'll add that back.

And you are correct the rest of the changes are just additions and cleaned-up.

Reply:
I looked up why Loliserver was removed, the project is dead and so are the download locations. It was replaced by Catserver. So there is no use in adding Loliserver back since even on an older version you'll get an error.

Reply:
Thanks!